scripts/bindata.sh

```diff
@@ -17,6 +17,11 @@ if [ -z "$GOPATH" ]; then
     export GOPATH=$(go env GOPATH)
 fi
 
+# Ensure the assets have the expected permissions before we package
+# them, in case the user's umask changes them from the desired
+# settings.
+find ./assets -name '*.yaml' | xargs chmod 0644
```

Comment:
Should we just update the umask instead of touching permissions on the file system?

Reply:
The files are already checked out by the time this script runs, so I don't think just changing the umask will help, will it?

Reply:
Ah, I thought umask is applied on permissions by bindata. If bindata "copies" permissions from the file system, not much we can do except resetting those.

Reply:
Yeah, the umask comes into account when the user checks the files out with git, but bindata takes what it reads from the filesystem.